tmt/steps/provision/podman.py

```diff
@@ -44,6 +46,26 @@ class PodmanGuestData(tmt.steps.provision.GuestData):
         metavar='NAME',
         help='Name or id of an existing container to be used.')
 
+    pull_attempts: Optional[int] = field(
+        default=DEFAULT_PULL_ATTEMPTS,
+        option='--pull-attempts',
+        metavar='ATTEMPT',
+        help=f"""
+             How many times to try pulling the image,
+             {DEFAULT_PULL_ATTEMPTS} attempts by default.
+             """,
+        normalize=tmt.utils.normalize_int)
+
+    pull_interval: Optional[int] = field(
```

Comment:
No need for `Optional`, there will always be a value as the field does have a non-`None` default.